azdev/config/cli_pylintrc

```diff
@@ -50,13 +54,13 @@ min-similarity-lines=10
 # The invalid-name checker must be **enabled** for these hints to be used.
 include-naming-hint=yes
 
-module-name-hint=lowercase (keep short; underscores are discouraged)
-const-name-hint=UPPER_CASE_WITH_UNDERSCORES
-class-name-hint=CapitalizedWords
-class-attribute-name-hint=lower_case_with_underscores
-attr-name-hint=lower_case_with_underscores
-method-name-hint=lower_case_with_underscores
-function-name-hint=lower_case_with_underscores
-argument-name-hint=lower_case_with_underscores
-variable-name-hint=lower_case_with_underscores
-inlinevar-name-hint=lower_case_with_underscores (short is OK)
+module-naming-style=snake_case
+const-naming-style=UPPER_CASE
+class-naming-style=PascalCase
+class-attribute-naming-style=snake_case
+attr-naming-style=snake_case
+method-naming-style=snake_case
+function-naming-style=snake_case
+argument-naming-style=snake_case
+variable-naming-style=snake_case
+inlinevar-naming-style=snake_case
```

Comment:
Any reason for the name change?

Reply:
I think this is a incompatibility introduced by pylint 2.14
> Several occurrences of E0015: Unrecognized option found on the config file: 
function-name-hint, variable-name-hint, const-name-hint, attr-name-hint, argument-name-hint, class-attribute-name-hint, inlinevar-name-hint, class-name-hint, module-name-hint, method-name-hint - This is where the incompatibility is: The old option names are rejected by pylint 2.14, and the new option names are rejected by older pylint versions.

Ref: https://github.com/pylint-dev/pylint/issues/6931

Reply:
I saw the document has been moved to https://pylint.pycqa.org/en/v2.17.4/user_guide/messages/convention/invalid-name.html#predefined-naming-styles

Reply:
After further investigation, these naming-style config can be removed.
1. They are nearly the same as default value.
2. `invalid-name` rule is disabled, these configurations make no sense.
3. `cli_pylintrc` hardly take effect as CLI has its own `pylintrc`.

I've removed them in CLI.
https://github.com/Azure/azure-cli/pull/26685#discussion_r1230661713